force-app/main/default/lwc/nksVideoPlayer/nksVideoPlayer.html

```diff
@@ -0,0 +1,40 @@
+<template>
+    <div if:true={isFileTypeMp4}>
+        <lightning-layout horizontal-align="center" multiple-rows>
+            <lightning-layout-item size="12" medium-device-size="10">
+                <div if:true={isExperience}>
+                    <div class="decorator-wrapper slds-p-bottom_small">
+                        <h1 class="navds-heading navds-heading--xlarge" style="text-align: center;">
+                            {videoTitle}
+                        </h1>
+                    </div>
+                </div>
+                <div style="position: relative">
+                    <div if:true={error}>
+                        <div class="decorator-wrapper alert-positioning">
+                            <div
+                                style="align-items: inherit"
+                                class="navds-alert navds-alert--error navds-alert--medium"
+                            >
+                                <c-icon icon-name="ErrorFilled" fill="Red"></c-icon>
+                                <div class="navds-alert__wrapper navds-body-long">
+                                    <h3 class="navds-heading navds-heading--small navds-typo--spacing">
+                                        {label.NKS_Video_Player_Error_Header}
+                                    </h3>
+                                    {label.NKS_Video_Player_Error_Message}
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                    <div style="text-align: center" if:true={videoSrc}>
+                        <video class="subtitle-background" height="40%" width="60%"
```

Comment:
Syntes det er veldig rart med denne height og width hardkodet i videoen. Forstår at det er veldig greit for deg å teste, men du må huske å rydde det opp før ihb er "ferdig"

Reply:
Det er for å begrense størrelsen internt i Salesforce